frontend/src/services/chats-api/chats-api.service.ts

```diff
@@ -71,6 +82,14 @@ class ChatsApi {
       },
     );
   }
+
+  public listenToNewMessages(cb: NewMessagesListener): void {
+    this.#socket.on(SocketEvent.MESSAGE, cb);
+  }
+
+  public removeMessageListener(): void {
+    this.#socket.off(SocketEvent.MESSAGE);
+  }
```

Comment:
We don't need this in chatsApi
```suggestion
```

Reply:
Vlad asked to move listeners to this service :thinking:

Reply:
We discussed this and decided to move this logic in middleware